packages/invoices/src/OutboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -72,35 +73,73 @@ const getStatusOptions = (
   return options;
 };
 
-export const StatusChangeButton = () => {
-  const { success } = useNotification();
-  const isDisabled = useIsOutboundDisabled();
-  const { status, update } = useOutboundFields('status');
-  const t = useTranslation('distribution');
+const getNextStatusOption = (
```

Comment:
I basically took all the things in the component and extracted it to the `useStatusChangeButton` because that makes me feel better.... then just added the call to `useConfirmationModal` and it works pretty well!